fix(curriculum): English editing (#37814)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-class-syntax-to-define-a-constructor-function.english.md

## Description
<section id='description'>

> ES6 provides new syntax to create objects, using the keyword <dfn>class</dfn>.

Reviewer: ```suggestion
ES6 provides new syntax to create objects, using the <dfn>class</dfn> keyword.
```

Being consistent with other such `dfn`/`code` statements.

Also, IMO, `"a new syntax"` was fine here.

var zeus = new SpaceShuttle('Jupiter');
```

> The <code>class</code> syntax simply replaces the constructor function:

Reviewer 2: @waldoluisribeiro I am thinking `creation` should be left in here.

const zeus = new SpaceShuttle('Jupiter');
```

It should be noted that the <code>class</code> keyword declares a new function, to which a constructor is added. This constructor is invoked when <code>new</code> is called to create a new object.<br>
<strong>Notes:</strong><br><ul>
<li> UpperCamelCase should be used by convention for ES6 class names, as in <code>SpaceShuttle</code> used above.</li>
<li> The constructor method is a special method for creating and initializing an object created with a class. You will learn more about it in the Object Oriented Programming section of the JavaScript Algorithms And Data Structures Certification.</li></ul>
</section>

## Instructions
<section id='instructions'>
Use the <code>class</code> keyword and write a constructor to create the <code>Vegetable</code> class.

> The <code>Vegetable</code> class allows you to create a vegetable object with a property <code>name</code>, which is passed to the constructor.

Reviewer 2: ```suggestion
The <code>Vegetable</code> class allows you to create a vegetable object with a property <code>name</code>that gets passed to the constructor.
```
